tests/testthat/tests_02_evaluate_better_models.R

```diff
@@ -15,29 +15,21 @@ csv_data <- data.table(read.csv(csv_data_path))
 csv_correct_data_path <- file.path("tests/data/best_models_table_tests.csv")
 csv_correct_data <- data.table(read.csv(csv_correct_data_path))
 
-test_that("Los resultados generados del código son correctos y siguen la guía de estilo:",
-    {
-        expect_equal(json_data, json_correct_data)
-        expect_equal(csv_data, csv_correct_data)
-        expect_equal(length(readLines(json_data_path)), 43)
-        expect_equal(length(readLines("data/processed/logistic_model_parameters.json")), 43)
-    }
-)
+test_that("Los resultados generados del código son correctos y siguen la guía de estilo:", {
+  expect_equal(json_data, json_correct_data)
+  expect_equal(csv_data, csv_correct_data)
+  expect_equal(length(readLines(json_data_path)), 43)
```

Comment:
Intentamos darle la misma importacnia a las pruebas como al código que las hará pasar. Así las mismas buenas constumbres de programación que aplicamos al código deberemos aplicar a las pruebas. 

Por favor encapsula (mete en una variable) el número mágico y la "ruda" expresión `length(readLines(json_data_path))`, para que se vea algo así:
```suggestion
  expect_equal(obtained_length, expected_length)
```
Creo que esas variables las puedes definir justo antes de la prueba, en la línea 17, pero si puedes hacerlo dentro del `test_that` quedará mejor. Es decir si se ve algo así:
```
test_that("Los resultados generados del código son correctos y siguen la guía de estilo:", {
  obtained_length <- length(readLines(json_data_path))
  expected_length <- 43
  expect_equal(json_data, json_correct_data)
```
mejor.